Agent for Windows won't compile on Ubuntu 23.04

## Build log

```
[  3%] Building CXX object CMakeFiles/sysinfo.dir/src/network/networkInterfaceWindows.cpp.obj
In file included from /root/wazuh-v4.7.1-rc2/src/data_provider/src/network/networkInterfaceWindows.cpp:13:
/root/wazuh-v4.7.1-rc2/src/shared_modules/utils/windowsHelper.h: In static member function 'static std::string Utils::NetworkWindowsHelper::getIpV6Address(const uint8_t*)':
/root/wazuh-v4.7.1-rc2/src/shared_modules/utils/windowsHelper.h:576:64: error: aggregate 'std::array<char, 16> buffer' has incomplete type and cannot be defined
  576 |                     std::array<char, IPV6_BUFFER_ADDRESS_SIZE> buffer;
      |                                                                ^~~~~~
/root/wazuh-v4.7.1-rc2/src/shared_modules/utils/windowsHelper.h:579:64: error: aggregate 'std::array<char, 16> addrComparator' has incomplete type and cannot be defined
  579 |                     std::array<char, IPV6_BUFFER_ADDRESS_SIZE> addrComparator;
      |                                                                ^~~~~~~~~~~~~~
make[4]: *** [CMakeFiles/sysinfo.dir/build.make:77: CMakeFiles/sysinfo.dir/src/network/networkInterfaceWindows.cpp.obj] Error 1
```

## Proposed fix

- Add this line in _windowsHelper.h_:

```c
#include <array>
```

--- a/src/shared_modules/utils/windowsHelper.h
+++ b/src/shared_modules/utils/windowsHelper.h
@@ -17,6 +17,7 @@
 #include <map>
 #include <memory>
 #include <vector>
+#include <array>
 #include <system_error>
 #include <winsock2.h>
 #include <windows.h>

